### DISCUSSION TITLE
Using Khoj with the Postgres DB on Remote Server

### DISCUSSION THREAD

**Discussion by Yi-Lyu** [khoj-ai/khoj#594] [Q&A] [answered]
Question, if my Postgres database is deployed on a remote server, how should I configure the database configuration when deploying it myself?

**Comment by sabaimran** [comment, marked as answer]
Hi @Yi-Lyu , the Khoj application takes in the following environment variables:
```
POSTGRES_DB
POSTGRES_USER
POSTGRES_PASSWORD
POSTGRES_HOST
```

By setting these, you can configure the connection between your Khoj server and your remote DB. If you're using Docker, you can directly update the environment variables. If you're using the local code or pypi package directly, you can set them however you configure your environment variables in your CLI.

**Comment by debanjum** [comment]
I've converted this into a Q&A discussion from the original issue
